homework/index.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+function main() {
+
+    const mainUrl = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+
+    // initial page: create header and main container.
+    const root = document.getElementById('root');
+    const header = createAndAppend('header', root);
```

Comment:
I like that you have used semantic HTML5 elements, such as `<header>`, `<main>` and `<section>` instead of just `<div>`'s.